src/diffpy/srmise/tests/test_dataclusters.py

```diff
@@ -32,3 +33,48 @@ def test___eq__():
             print(f"not-equal test failed on {attr_key}")
             assert not expected == actual
         attributes.update({attr_key: reset})
+
+
+@pytest.mark.parametrize(
+    "inputs, expected",
+    [
+        (
+            {
+                "x": np.array([1, 2, 3]),
+                "y": np.array([3, 2, 1]),
+                "res": 4,
+            },
+            DataClusters(np.array([1, 2, 3]), np.array([3, 2, 1]), 4),
```

Comment:
This doesn't appear to be testing anything as your actual and expected are both just running the same functions.